### ISSUE TITLE
wrong file extension (.py instead of .pyx) produces misleading error message

### ISSUE THREAD

**Issue by Phillip-M-Feldman** [cython/cython#1824]
If a Cython file has the wrong extension--.py instead of .pyx, the Cython compiler generates misleading syntax errors when it encounters Cython keywords, as shown below:

```
>Cython bug.pyx

>rename bug.pyx bug.py

>Cython bug.py

Error compiling Cython file:
-------------------------------------------------
...
cdef int add(int x, int y):
    ^
-------------------------------------------------

bug.py:1:5: Syntax error in simple statement list
```

Is it possible to get a more helpful error message, along the lines of the following?  "Use of Cython keywords is not permitted in pure Python source files."

**Comment by Phillip-M-Feldman** [comment]
An explanation for why this issue was summarily closed would be appreciated.

**Comment by scoder** [comment]
It was closed automatically by committing a fix: 7f9c0dcaa317dd0b90d74730aa08de3278e1096f

**Comment by Phillip-M-Feldman** [comment]
That's perfect!
